ffig/FFIG.py

```diff
@@ -78,14 +78,47 @@ def build_model_from_source(path_to_source, module_name):
     Returns:
     - model built from a clang.cindex TranslationUnit with a name from args
     """
-    tu = clang.cindex.TranslationUnit.from_source(
-        path_to_source, '-x c++ -std=c++14 -stdlib=libc++'.split())
+    try:
+        tu = clang.cindex.TranslationUnit.from_source(
+            path_to_source, '-x c++ -std=c++14 -stdlib=libc++'.split())
+    except TranslationUnitLoadError:
```

Comment:
Seems that upstream libclang's (and our fork) TranslationUnitLoadError is a TODO. 
https://github.com/FFIG/ffig/blob/master/ffig/clang/cindex.py#L120

re: testcases, do I understand that I need a bad Cpp fail to guarantee it will fail clang_parseTranslationUnit?